docker/thingd/config.conf

```diff
@@ -0,0 +1,83 @@
+# This is an example file of how this configuration file parameters should be
```

Comment:
Could we add these comments in a documentation, under `/docs`, and go over each parameter? This documentation could be referenced in `README.md`. 

```markdown
# Thingd Configuration File 

A very brief of what [Thingd](link-to-repo) is and why we need it in the webui. 

## KNoT Thing Parameters

...

## KNoT Data Items Parameters

....

```

If other tasks are being prioritized, let me know so I can make some remarks to improve the comments in this file instead :smiley:

Reply:
I think this documentation should be referenced from the _thingd_ repo. So, I wouldn't pay an effort to create a well-formatted markdown here since this will be used only to simulate the `devices.conf` file. However, it would be good to document the `backend` endpoints as I started [here](https://github.com/CESARBR/knot-gateway-webui/tree/backend-next/backend/api) in the golang migration branch. 

I think @Igoraamc could create this documentation under `/docs` (as @lcbm suggested) and add the endpoint being implemented in this PR (POST `/devices`).